deb/common/docker-ce.postinst

```diff
@@ -1,13 +1,36 @@
 #!/bin/sh
 set -e
 
+update_dockerd() {
+	dbefile=/var/lib/docker/distribution_based_engine.json
+	URL=https://docs.docker.com/releasenote
+	if [ -f "${dbefile}" ] && sed -e 's/.*"platform"[ \t]*:[ \t]*"\([^"]*\)".*/\1/g' "${dbefile}"| grep -v -i community > /dev/null; then
+		echo
+		echo
+		echo
+		echo "Warning: Your engine has been activated to Docker Engine - Enterprise but you are still using Community packages"
+		echo "You can use the 'docker engine update' command to update your system, or switch to using the Enterprise packages."
+		echo "See $URL for more details."
+		echo
+		echo
+		echo
+	else
+		update-alternatives --install /usr/bin/dockerd dockerd /usr/bin/dockerd-ce 1 --slave \
```

Comment:
Just noticed this didn't get the "--force" flag added to it from my last PR.

Reply:
The `--force` flag doesn't exist on RPM based distributions so I decided to keep it uniform amongst the two.

Is there a need for a `--force` flag?

Reply:
Interesting... 

On Ubuntu my testing of the CE: install/update/patch flow required it to work properly.

Reply:
Instead of using force, we can instead remove the old link explicitly with an `rm -f  /usr/bin/dockerd` just before calling `update-alternatives`